Makefile

```diff
@@ -27,10 +19,10 @@ lint:
 
 .PHONY: frontend
 frontend:
-	go build -ldflags "-X main.Version=$(VERSION)" -o aro-hcp-frontend ./frontend
```

Comment:
I'm fine with removing `VERSION` but can you replace the `Version` variable in `frontend/main.go`?  Maybe pass the commit hash like `-ldflags "-X main.Commit=$(COMMIT)".  This is just so _some_ kind of revision identifier is logged at startup.

Reply:
Oh the revision identifier was somewhat replaced by this chunk in `frontend/main.go` - I'll add this to the PR description

```go
	version := "unknown"
	if info, ok := debug.ReadBuildInfo(); ok {
		for _, setting := range info.Settings {
			if setting.Key == "vcs.revision" {
				version = setting.Value
				break
			}
		}
	}
```

Reply:
Ok cool, that works for me.